apps/catalogue/src/views/HarmonizationRow.vue

```diff
@@ -42,10 +42,11 @@ export default {
       }, {});
     },
     getMatchStatus(resource) {
-      if (this.variable.repeats) {
+      if (this.variable.repeats && Array.isArray(this.resourceMappings)) {
         const statusList = this.variable.repeats.map((repeatedVariable) => {
           const resourceMapping = this.resourceMappings.find((mapping) => {
             return (
+              mapping.targetVariable &&
```

Comment:
use ? instead

Reply:
then you would return 'false' right instead of the desired output?

Reply:
no , i think the find evaluates the function for each row and return the fist row for which te function evaluates true , if the row -> func -> false is continues to the next row

Reply:
ok, lets see